src/app/page.tsx

```diff
@@ -1,51 +1,23 @@
-import { Button } from "@mantine/core";
-import QueryWrapper from "./query/wrapper";
+import { mockEmptyIssue, mockIssue, mockUser } from "@/mock/conversationMock";
+import HomePageCard from "../components/HomePage/HomePageCard";
+import type { Metadata } from "next";
 
+export const metadata: Metadata = {
+    title: "CommonGround",
+    description: "A platform for people to discuss and share their opinions.",
+};
+
+const issues = [mockIssue, mockEmptyIssue];
+const user = mockUser;
```

Comment:
Same for the user, as the user object will be retrieved using the `useSession` hook